cypresse2e/cypress/elements/pages/aboutUs.js

```diff
@@ -0,0 +1,30 @@
+class AboutUs {
+  linkedInButton = "#AU_LN";
+  facebookButton = "#AU_FB";
+  instagramButton = "#AU_IG";
+  twitterButton = "#AU_TW";
+  linkedInLink = "https://www.linkedin.com/company/fightpandemics/";
+  facebookLink = "https://www.facebook.com/FightPandemics/";
+  instagramLink = "https://www.instagram.com/fightpandemics/";
+  twitterLink = "https://twitter.com/FightPandemics";
+  getInvolvedCloseButton = ".ant-modal-close";
+  getInvolvedButton = "#AU_GI";
+  joinAsVolunteerButton = "#AU_GI_JV";
+  joinAsVolunteerLink = "https://apply.workable.com/fightpandemics";
+  joinAsAmbassadorButton = "#AU_GI_JA";
+  joinAsAmbassadorLink = "https://apply.workable.com/fightpandemics/j/46D6EF3B44/";
+  joinStudentProgramButton = "#AU_GI_JSP";
+  joinStudentProgramLink = "https://apply.workable.com/fightpandemics/j/58B157AAB2/";
+  getToHelpBoardButton = "#AU_VHB";
+  goToHelpBoardLink = "/feed";
+  partnershipEmail = "partnerships@fightpandemics.com";
+  contactEmail = "contact@fightpandemics.com";
+  partnershipEmailHref = "mailto:partnerships@fightpandemics.com";
+  contactEmailHref = "mailto:admin@fightpandemics.com";
```

Comment:
CHange to contact@@fightpandemics.com

Reply:
@TimeaKovacs86 

I was thinking also about moving some links to fixtures (like: social media, emails). They can be used across tests.

Reply:
Yea @galeza this is a good idea. A json file named something like socialContactInfo can be created in the fixtures folder.

Reply:
Done, created a socialContactInfo.json file in the fixtures directory, which contains the social media links and the partnership and contact emails. It goes up with the next commit!